CHANGELOG.en-US.md

```diff
@@ -15,6 +15,12 @@ tag: vVERSION
 
 ---
 
+## 5.26.2
+
+`2025-06-20`
```

Comment:
```suggestion
`2025-06-21`
```

Reply:
i will update to 06-22, sorry i miss the message today.